### ISSUE TITLE
Cannot install any extensions in Release 1.75.1

### ISSUE THREAD

**Issue by andrewhaine1** [microsoft/vscode#175556]
Type: <b>Bug</b>

Update to Release 1.75.1, no extensions can be installed.
Downgraded to 1.74, same thing.
Downgraded to 1.73, everything back to normal.

Error:
2023-02-26 18:46:45.612 [warning] Via 'product.json#extensionEnabledApiProposals' extension 'github.vscode-pull-request-github' wants API proposal 'commentsResolvedState' but that proposal DOES NOT EXIST. Likely, the proposal has been finalized (check 'vscode.d.ts') or was abandoned.
2023-02-26 18:46:46.688 [error] Error scanning installed extensions:
2023-02-26 18:46:46.688 [error] SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at Object.factory (vscode-file://vscode-app/c:/Program%20Files/Microsoft%20VS%20Code/resources/app/out/vs/workbench/workbench.desktop.main.js:741:99212)

Extension version: 2023.2.0
VS Code version: Code 1.73.1 (6261075646f055b99068d3688932416f2346dd3b, 2022-11-09T04:27:29.066Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7300U CPU @ 2.60GHz (4 x 2712)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.87GB (5.91GB free)|
|Process Argv|--crash-reporter-id d49c0d37-6afd-491f-907a-ca9e1d6cc2dc|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vsdfh931cf:30280410
vshan820:30294714
vstes263:30335439
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012cf:30540253
azure-dev_surveyone:30548225
vsccc:30610678
pyindex848:30662994
nodejswelcome1cf:30587006
282f8724:30602487
pyind779:30671433
89544117:30613380
pythonsymbol12:30671437
2i9eh265:30646982
pythonb192:30669360

```

</details>

**Comment by trandangquyen** [comment]
Thanks [andrewhaine1](https://github.com/andrewhaine1). I have tested and quick to fix this issue is downgrade to VSCodeSetup-x64-1.73.1 version. Both version 1.74 and 1.75 appear above error like you said

**Comment by sandy081** [comment]
It shall be fixed in the latest `1.76.0` release - Please update to latest and let us know if you are still seeing it.
